README.Docker.md

```diff
@@ -0,0 +1,246 @@
+# ClamAV in Docker
```

Comment:
Some markdown editor linters will complain that you need a new line after the heading. It will also match our other markdown

Reply:
Ah sorry, yeah; I know that this behavior stems from 'at some point, there was a broken parser, so lets fix the linter/user behavior'
Very annoying. For rendered MD it makes no sense anyway, so who cares, and for people reading it in a text editor, its much friendlier to read.

Oh well :) i'll match your MD